src/util/loadExternals.ts

```diff
@@ -52,6 +54,7 @@ function networkLoad(
   const request = new XMLHttpRequest();
   const loadedFunction = function(this: XMLHttpRequest): void {
     external.content = this.responseText;
+    external.failed = this.status >= 299;
```

Comment:
This should be >= 400 since 300 status codes are redirected. I don't think we'll ever land here with status codes in the 300 range, but it's not technically an error.

Reply:
Agree.